rackcity/views/asset_views.py

```diff
@@ -245,6 +268,82 @@ def save_mac_addresses(asset_data, asset_id):
         raise MacAddressException(failure_message)
 
 
+def save_network_connections(asset_data, asset_id):
```

Comment:
What will happen first? 1000 commits or 1000 lines in asset_views? Flagged on the last PR, I think this and the mac/power equivalents should live elsewhere, but I'm not convinced a better place currently exists